applications/graphql/main.tf

```diff
@@ -97,20 +97,41 @@ module "app" {
   }
 }
 
+module "contentful_webhook" {
+  source = "../../components/lambda_function"
+
+  name    = "${var.name}-webhook"
+  handler = "webhook.handler"
+  runtime = "nodejs8.10"
+  logger  = "${var.logger}"
+
+  config_vars = {
+    NODE_ENV       = "production"
+    QUERY_ENV      = "${local.env}"
+    CACHE_DRIVER   = "dynamodb"
+    DYNAMODB_TABLE = "${module.cache.name}"
+  }
+}
+
 module "gateway" {
   source = "../../components/api_gateway"
 
   name   = "${var.name}"
   domain = "${var.domain}"
 
-  functions     = ["${module.app.arn}"]
+  functions     = ["${module.app.arn}", "${module.contentful_webhook.arn}"]
```

Comment:
what's an `arn`? 🤔

Reply:
Amazon's [silly name for IDs](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html). 🤷‍♂️

Reply:
ah! 🤪